README.md

```diff
@@ -1 +1,42 @@
-# taxi
+#  🚖 **Taxi service**
+The application describes the operation of the taxi service. </br>
+To implement the application were created: </br>
+- taxi_service database
+- models: Car, Driver, Manufacturer
+- interfaces DAO and implements for each model
+- interfaces "Services" and implements for each model and authentication
+- logger in the AuthenticationService
+- filter for authentication driver
+- controllers for all actions
```

Comment:
let's move this info to the paragraph `Implementation details and technologies` and add this info not as separate paragraphs but as text info, and make it make it shorter